charger/easee.go

```diff
@@ -424,12 +426,21 @@ func (c *Easee) Enable(enable bool) error {
 	if err != nil {
 		return err
 	}
-	if noop {
-		return c.confirmChargerCurrent(targetCurrent)
-	}
 
-	if err := c.waitForDynamicChargerCurrent(targetCurrent); err != nil {
-		return err
+	if noop {
+		if err := c.confirmChargerOpMode(targetOpMode); err != nil {
```

Comment:
Brauchen wir hier die beiden `confirm` Funktionen mit nur jeweils einer Anwendung und 2x Lock? Wäre einfacher das hier direkt inline zu machen? Oder in eine Funktion kombinieren?

Reply:
Kombinieren geht leider nicht, weil wir `confirmChargerCurrent()` auch in `MaxCurrent()` benötigen.
Wir könnten es inlinen - dann sollten wir aber confirmChargerCurrent() auch in MaxCurrent() inlinen und komplett entsorgen.